src/apis/platformvm/utxos.ts

```diff
@@ -88,18 +89,31 @@ export class UTXO extends StandardUTXO {
   }
 
   /**
-   * Takes a base-58 string containing a [[UTXO]], parses it, populates the class, and returns the length of the StandardUTXO in bytes.
+   * Takes a base-58 or hex string containing a [[UTXO]], parses it, populates the class, and returns the length of the StandardUTXO in bytes.
    *
    * @param serialized A base-58 string containing a raw [[UTXO]]
+   * @param format The format of the encoded [[UTXO]] (cb58 or hex). Defaults to cb58 per existing codebase
    *
    * @returns The length of the raw [[UTXO]]
    *
    * @remarks
    * unlike most fromStrings, it expects the string to be serialized in cb58 format
    */
-  fromString(serialized: string): number {
-    /* istanbul ignore next */
-    return this.fromBuffer(bintools.cb58Decode(serialized))
+  fromString(serialized: string, format: string = 'cb58'): number {
+    switch (format) {
+      case "cb58": {
+         /* istanbul ignore next */
+        return this.fromBuffer(bintools.cb58Decode(serialized));
+      };
+      case "hex": {
```

Comment:
Please add test coverage for `"hex"` case.

Reply:
Added, if not enough please shout and I can add more.